extensions/amp-wordpress-embed/OWNERS.yaml

```diff
@@ -0,0 +1,2 @@
+- westonruter
+- amedina
```

Comment:
Please use the JSON format for all new OWNERS files. This will become:

```javascript
// For an explanation of the OWNERS rules and syntax, see:
// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example

{
  rules: [
    {
      owners: [
        { name: "westonruter" },
        { name: "amedina" }
      ]
    }
  ]
}
```

Reply:
Also, /cc @kristoferbaxter in case he thinks we should add `ampproject/wg-ui-and-a11y` as an owner